services/ResumeService.ts

```diff
@@ -0,0 +1,24 @@
+import { ResumeModel } from 'models/ResumeModel';
+import { NotFoundError } from 'routing-controllers';
+import { Service } from 'typedi';
+import { EntityManager } from 'typeorm';
+import { InjectManager } from 'typeorm-typedi-extensions';
+import Repositories, { TransactionsManager } from '../repositories';
+// import { ResumeRepository } from '../repositories/ResumeRepository';
+
+@Service()
+export default class ResumeService {
+  private transactions: TransactionsManager;
+
+  constructor(@InjectManager() entityManager: EntityManager) {
+    this.transactions = new TransactionsManager(entityManager);
+  }
+
+  public async getVisibleResumes(canSeeAllVisibleResumes = false) : Promise<ResumeModel[]> {
+    if (!canSeeAllVisibleResumes) return [];
+    const resumes = await this.transactions.readOnly(async (txn) => Repositories
```

Comment:
Related to my first comment, let's throw on permission error instead of returning an empty response